integration/conftest.py

```diff
@@ -1,14 +1,69 @@
 import os
+from datetime import datetime
 
 import pytest
 
 import py42.sdk as _sdk
 
-HOST_URL = "https://console.us.code42.com"
+
+def pytest_addoption(parser):
```

Comment:
This looks like `adoption` with two `d`s... Can say `add_option` ?

Reply:
This is a pytest hook, and the name has to be `pytest_addoption`. Otherwise it won't read the config file.

Reply:
Can we check in an ini file with these values blank so it's clear where to put it?

Maybe also add a section to the contributing guide about integration tests, how to run them, how to skip them, etc.

Reply:
- All values are already added to ini file. 

- Also the change to skip integration tests is also covered in ini file. So right now unit test execution will behave as it behaves in master. i.e integration tests will be skipped.

-I will make a note in contributinging guide to add username and password in appropriate environment variables.

Reply:
Ah, sorry. I was looking for a pytest.ini file, and hadn't yet got to the tox.ini file part of the review.

Reply:
added notes to contributing guide.